### PR TITLE
[ADD] doc: from community to enterprise

### PR DESCRIPTION
**Pull request by mgeubelle**

### CODE REVIEW

File: doc/setup/enterprise.rst [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+  .. code-block:: console
+
+    $ sudo service odoo stop
+
+* Install the enterprise .deb (it should install over the community package)
+
+  .. code-block:: console
+
+    $ sudo dpkg -i <path_to_enterprise_deb>,
```
Comment (dbo-odoo) [inline doc/setup/enterprise.rst]: remove ,

File: doc/setup/enterprise.rst [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+  .. code-block:: console
+
+    # --addons-path=~/src/custom_modules,~/src/enterprise,~/src/odoo/addons
+
+* Update the web module by using
+
+  .. code-block:: console
+
+    # -d <database_name> -u web --stop-after-init.
```
Comment (dbo-odoo) [inline doc/setup/enterprise.rst]: remove .

File: doc/setup/enterprise.rst [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+On Linux, using the source code
+===============================
+
+There are many ways to launch your server when using sources, and you probably
+have your own favourite. You may need to adapt sections to your usual workflow.
+
+* Shutdown your server
+* Backup your community database
+* Update the --addons-path parameter of your launch command (init.d, custom script,
+  configuration file, etc.). The Enterprise addons folder should be included ``before`` the community modules.
```
Comment (dbo-odoo) [inline doc/setup/enterprise.rst]: use **double stars** \*\* to add emphasis instead of code braces

File: doc/setup/enterprise.rst [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+  For example:
+
+  .. code-block:: console
+
+    # --addons-path=~/src/custom_modules,~/src/enterprise,~/src/odoo/addons
```
Comment (dbo-odoo) [inline doc/setup/enterprise.rst]: replace the # by a $ otherwise some people may assume you need to do this as root

**Comment by dbo-odoo** [comment]
Some remarks: try to use code delimiters (\`) when appropriate, like for `init.d` or when mentionning the `--addons-path` parameter
Check if the flow is ok for rpm files as well (i think the dpkg command only works on .deb)
Don't hesitate to add emphasis, warnings (yellow panels), etc. (check other rst files to see the syntax)

**Comment by xmo-odoo** [comment]
@dbo-odoo technically rST/Sphinx have a bunch of more semantically weighty roles for inline elements than just code e.g. [`:file:`](http://www.sphinx-doc.org/en/stable/markup/inline.html#role-file) or [`:option:`](http://www.sphinx-doc.org/en/stable/markup/inline.html#role-option) (which should cross-link to the corresponding options documentation if there is one)

**Pull request closed without merging**